qtranslate.php

```diff
@@ -54,7 +54,7 @@
 	ja by Brian Parker
 	nl by RobV
 	pt by netolazaro
-	pt-br by Pedro Mendonça
+	pt-pt by Pedro Mendonça
```

Comment:
I meant you introduced "pt-br" to -X, not to the World ;) I am confused, because there is no "pt-pt" in -X, there are only "pt" and "pt-br"